src/containers/CharacterProfile/index.js

```diff
@@ -54,19 +59,27 @@ class CharacterProfileScreen extends Component {
               <View style={Styles.backDrop}/>
               <ProfilePicture image='./../../img/headshots/Tile-Kazuya.png' />
               <ProfileName name={characterID.toUpperCase()} />
-              <MoveList moves={moves} />
+            <MoveList moves={filtered} />
          </ScrollView>
       </View>
       </SideMenu>
     );
   }
 }
 
+function filteredAttacks(moves) {
+  let attackFilters = this.props.filter;
+  return moves.filter(attack => attackFilters.every(filter => filter(attack)));
+}
 
 /** MAPPING STATE **/
 const mapStateToProps = function(state) {
+  console.log(state);
+  let {filter, character } = state;
```

Comment:
@ndejesus1227  Is the filter property on the top level of the store state? We should move it into the character state, since thats what this filter is for.